jupyterhub/user.py

```diff
@@ -107,11 +116,7 @@ def _orm_user_changed(self, change):
         else:
             self._user_id = None
 
-    spawner = None
-    spawn_pending = False
-    stop_pending = False
-    proxy_pending = False
-    waiting_for_response = False
+    spawners = None
```

Comment:
Any reason not to move this statement up to line 92 to follow `settings = Dict()`? It's a bit hard to distinguish at quick glance if this is part of `_db_changed`. It's not.